terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/main/java/org/terasoluna/gfw/web/exception/ExceptionLoggingFilter.java

```diff
@@ -159,7 +159,7 @@ protected void logServletException(ServletException ex,
      */
     protected void logRuntimeException(RuntimeException ex,
             ServletRequest request, ServletResponse response) {
-        exceptionLogger.log(ex);
+        exceptionLogger.error(ex);
     }

```

Comment:
@yoshikawaa , @kazuki43zoo

I think that it is not appropriate to delete these logging methods because the `ExceptionLoggingFilter` class is extensible design and there is a possibility that there are projects that has already been expanded and used.